Added prizes and updated tracks

kjustin2 opened the pull request:
Only issue is ensuring that we have the prizes set in stone for the 1st, 2nd, and 3rd place positions. Currently these prizes are the same as last year.

RitwikGupta commented on index.html:
```diff
@@ -203,8 +204,33 @@ <h2>Challenges</h2>
+                                        <header class="major">
+                                            <h2>Tracks</h2>
+                                        </header>
+										<h3><strong>Choose a track (or combine multiple!) to focus your idea upon.</strong></h3>
```
The indentation here and at line 218 onwards is really messed up. Fix this and then we can merge.

kjustin2 replied:
Well that's odd, something must have changed my indentation during upload.
I'll hop on fixing that up.

RitwikGupta commented:
Reverted. Just realized this breaks the tracks section. No videos come up.

kjustin2 commented:
Hello,


The videos don't come up because we do not have videos for these tracks yet.


Those videos that are not being shown are from last year's event, and do not

reflect the changes made in this year's competition.


Thanks,


Justin Kramer

________________________________
From: Ritwik Gupta <notifications@github.com>
Sent: Sunday, January 13, 2019 4:20:03 PM
To: PittChallenge/pittchallenge.com
Cc: Kramer, Justin P; Author
Subject: Re: [PittChallenge/pittchallenge.com] Added prizes and updated tracks (#25)


Reverted. Just realized this breaks the tracks section. No videos come up.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FPittChallenge%2Fpittchallenge.com%2Fpull%2F25%23issuecomment-453866552&data=02%7C01%7Cjpk91%40pitt.edu%7Cd7657fe752c74e9d5c0808d6799ce258%7C9ef9f489e0a04eeb87cc3a526112fd0d%7C1%7C0%7C636830112061209461&sdata=VUO8J%2Fmc%2FBkdf3hz7hIrRbjSzjyMt66P9YzU3mwwkAA%3D&reserved=0>, or mute the thread<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAezkTdEE7e4rDs3llUG5ObrTe1v5jD56ks5vC6MDgaJpZM4Z8TO4&data=02%7C01%7Cjpk91%40pitt.edu%7Cd7657fe752c74e9d5c0808d6799ce258%7C9ef9f489e0a04eeb87cc3a526112fd0d%7C1%7C0%7C636830112061209461&sdata=9zMYbhuXd2icA%2FzM2S7cs8OkKvntdBgoPGqwXnFo1Rk%3D&reserved=0>.

RitwikGupta commented:
Fair, reverted the revert 😁
